src/SearchRequestAdapter.js

```diff
@@ -443,6 +443,20 @@ export class SearchRequestAdapter {
       typesenseSearchParams.vector_query = params.typesenseVectorQuery;
     }
 
+    // Natural language search parameters
+    if (params.nl_query) {
+      typesenseSearchParams.nl_query = params.nl_query;
+    }
+    if (params.nl_model_id) {
+      typesenseSearchParams.nl_model_id = params.nl_model_id;
+    }
+    if (params.nl_query_debug) {
+      typesenseSearchParams.nl_query_debug = params.nl_query_debug;
+    }
+    if (params.nl_query_prompt_cache_ttl) {
+      typesenseSearchParams.nl_query_prompt_cache_ttl = params.nl_query_prompt_cache_ttl;
+    }
+
```

Comment:
Any reason you have this code block? 

`params` is the instantsearch parameters, so it won't have any `nl_*` parameters coming from it.